### PR TITLE
[WIP][feature]removre jersey

### PR DESCRIPTION
**Pull request by huangKai-2323**
### What is the purpose of the change

removre jersey

### Related issues/PRs

Related issues: #590

### Brief change log

- Define the core abstraction and interfaces of the EngineConn Factory;
- Define the core abstraction and interfaces of Executor Manager.

### Checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests).
- [ ] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [Linkis mailing list](https://linkis.apache.org/community/how-to-subscribe) first)
- [ ] **If this is a code change**: I have written unit tests to fully verify the new behavior.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -414,36 +414,36 @@
...
-      <dependency>
-        <groupId>com.sun.jersey</groupId>
-        <artifactId>jersey-json</artifactId>
-        <version>${jersey.version}</version>
-      </dependency>
-      <dependency>
-        <groupId>com.sun.jersey.contribs</groupId>
-        <artifactId>jersey-guice</artifactId>
-        <version>${jersey.version}</version>
-      </dependency>
```
Comment (jackxu2011) [inline pom.xml:446]: this lines can't be deleted, the eureka server is still use

File: linkis-commons/linkis-module/pom.xml [hunk 2]
```diff
@@ -361,20 +361,20 @@
...
+    <!--        <exclusion>-->
+    <!--          <groupId>javax.ws.rs</groupId>-->
+    <!--          <artifactId>jsr311-api</artifactId>-->
+    <!--        </exclusion>-->
+    <!--        <exclusion>-->
+    <!--          <groupId>com.sun.jersey</groupId>-->
+    <!--          <artifactId>jersey-core</artifactId>-->
+    <!--        </exclusion>-->
+    <!--      </exclusions>-->
+    <!--    </dependency>-->
```
Comment (jackxu2011) [inline linkis-commons/linkis-module/pom.xml:377]: just remove the dependencies

File: linkis-commons/linkis-module/pom.xml [hunk 3]
```diff
@@ -328,24 +328,24 @@
...
+    <!--        <exclusion>-->
+    <!--          <groupId>javax.validation</groupId>-->
+    <!--          <artifactId>validation-api</artifactId>-->
+    <!--        </exclusion>-->
+    <!--      </exclusions>-->
+    <!--    </dependency>-->
+    <!--    <dependency>-->
+    <!--      <groupId>com.sun.jersey</groupId>-->
+    <!--      <artifactId>jersey-servlet</artifactId>-->
+    <!--    </dependency>-->
```
Comment (jackxu2011) [inline linkis-commons/linkis-module/pom.xml:348]: just remove the lines

**Review by jackxu2011 — CHANGES_REQUESTED** [review]
the eureka client alse have jersey, should be excluded.

File: linkis-commons/linkis-module/pom.xml (hunk 3 above)
Comment (huangKai-2323) [reply linkis-commons/linkis-module/pom.xml:348]: OK ,I  will fix it

File: linkis-commons/linkis-module/pom.xml (hunk 2 above)
Comment (huangKai-2323) [reply linkis-commons/linkis-module/pom.xml:377]: done

File: pom.xml (hunk 1 above)
Comment (huangKai-2323) [reply pom.xml]: OK ,I will fix it

**Comment by jackxu2011** [comment]
spring-cloud-starter-netflix-eureka-client still have jersey dependency

**Pull request closed without merging**